### PR TITLE
Move the EN translations in the code

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/models.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ class User(Base):
     id = Column(Integer, primary_key=True)
     username = Column(
         Unicode, unique=True, nullable=False,
-        label=_(u"Username")
+        label=_(u"User name")
```
Comment (asaunier) [inline c2cgeoportal/models.py]: Username is a single word

Comment (sbrunner) [inline c2cgeoportal/models.py]: Fixed, thanks :-)

**Comment by asaunier** [comment]
Just curious why you do this change since the translating is done anyway?

**Comment by sbrunner** [comment]
Because in Transifex we see the key and we should see the source language :-)

**Pull request merged**